configs/README.md

```diff
@@ -0,0 +1,129 @@
+feature conditioning.
+
+-   [`attentive_gru.yaml`](attentive_gru.yaml) and
+    [`attentive_lstm.yaml`](attentive_lstm.yaml) are attentive RNN models with
+    hyperparameters similar to those of MED (Kann & Schütze 2016).
+-   [`attentive_unilstm.yaml`](attentive_unilstm.yaml) is a variant of
+    [`attentive_lstm.yaml`](attentive_lstm.yaml) with a uni- (rather than bi-)
+    directional encoder. Performance is often quite a bit worse with this
+    design.
+-   [`attentive_gru2lstm.yaml`](attentive_gru2lstm.yaml) is a MED-like variant
+    where the encoder is an GRU but the decoder is a LSTM.
+-   [`attentive_deep_lstm.yaml`](attentive_deep_lstm.yaml) is a MED-like variant
+    where the encoder has 2 LSTM layers and the decoder has 2 LSTM layers.
+    Performance is often quite a bit worse with this design.
+-   [`frankenformer.yaml`](frankenformer.yaml) is a hybrid system with a
+    transformer encoder and an attentive LSTM decoder. This model is in need of
+    hyperparameter tuning before it is ready for deployment.
+-   [`hard_attention_lstm.yaml`](hard_attention_lstm.yaml) and
+    [`context_hard_attention_lstm.yaml`](context_hard_attention_lstm.yaml) are
+    variants of hard attention LSTMs with hyperparameters similar to those of Wu
+    & Cotterell (2019).
+-   [`pointer_generator_lstm.yaml`](pointer_generator_lstm.yaml) shows a
+    pointer-generator LSTM with hyperparameters similar to those of Sharma et
+    al. (2018, their "high setting").
+-   [`pointer_generator_transformer.yaml`](pointer_generator_transformer.yaml)
+    shows a pointer-generator transformer with hyperparameters similar to those
+    of Singer & Kann (2020).
+-   [`transducer_lstm.yaml`](transducer_lstm.yaml) shows a transducer LSTM with
+    hyperparameters similar to those used by Makarov & Clematide (2021).
+-   [`transformer.yaml`](transformer.yaml) is a transformer with hyperparameters
+    similar to those of Wu et al. (2021, "A smaller Transformer").
+
+## Feature-conditioned configurations
+
+Most of the patterns from above generalize to feature-conditioned transduction,
+but we illustrate different ways one can enable feature conditioning.
+
+-   [`attentive_lstm_shared.yaml`](attentive_lstm_shared.yaml) is a MED-like
+    variant with a shared source and features encoder.
+-   [`attentive_lstm_separate.yaml`](attentive_lstm_separate.yaml) is a MED-like
+    variant with a separate LSTM features encoder.
+-   [`attentive_lstm_linear.yaml`](attentive_lstm_linear.yaml) is a MED-like
+    variant with a separate linear features encoder.
+-   [`transformer_shared.yaml`](transformer_shared.yaml) is a transformer with a
+    shared source and features encoder.
+
+## Putting it all together
+
+[`algother.yaml`](altogether.yaml) is a single combined configuration file with
+trainer, checkpoint, data, model, and prediction configuration information.
+
+To use, run:
+
+    yoyodyne fit --config /path/to/config.yaml
```

Comment:
Can you point me to an example line for inference post trainign?

Reply:
See [here](https://github.com/kylebgorman/yoyodyne/tree/lightning?tab=readme-ov-file#inference-predict) in README for that:

```
yoyodyne predict --config path/to/config.yaml --ckpt_path path/to/checkpoint.ckpt
```